Biahun_Uladzimir/02.Git.local/02.Git.local

```diff
@@ -0,0 +1,67 @@
+      git init
```

Comment:
Where is markup here? Could you please use at least code section for your commands (add bash syntax highlighting)

Reply:
```
Новая версия проекта
```